src/providers/sms/bulutfon.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import Request from 'request-promise-native'
+// Types
+import type {SmsRequestType} from '../../models/notification-request'
+
+export default class BulutfonProvider {
+  id: string
+  config: Object
+
+  constructor (config: Object) {
+    this.id = 'sms-bulutfon-provider'
+    this.config = {
+      defaultTitle: config.defaultTitle,
```

Comment:
I think `defaultTitle` is not necessary in the provider configuration. This can be done in the service calling this lib (something like `from || 'default title'`). I would remove it.

Then `this.config` can be renamed `this.credentials`.